eras/shelley/test-suite/test/Test/Cardano/Ledger/Shelley/Examples/Mir.hs

```diff
@@ -243,15 +244,12 @@ mirFailWits pot =
     (initStMIR (Coin 1000))
     (blockEx1' insufficientMIRWits pot)
     ( Left
-        [ BbodyFailure @(ShelleyEra c)
-            ( LedgersFailure
-                ( LedgerFailure
-                    ( UtxowFailure $
-                        MIRInsufficientGenesisSigsUTXOW ws
-                    )
-                )
-            )
-        ]
+        . NE.singleton
+        . BbodyFailure @(ShelleyEra c)
+        . LedgersFailure
+        . LedgerFailure
+        . UtxowFailure
```

Comment:
Let's simplify this one a bit (thanks to the recent change that was merged that I was talking about during the meeting)
```suggestion
        . injectFailure
```